torch/distributed/_shard/sharded_tensor/metadata.py

```diff
@@ -22,6 +25,7 @@ class TensorProperties:
     pin_memory: bool = False
 
     def __getstate__(self):
+        """Get state of a shared tensor, unless in an invalid format."""
```

Comment:
Please remove the new dosctrings and only fix the errors listed in the issue. I think the issues clearly asks to skip the "Missing docstring" errors.